profiles/templates/profiles/profile.html

```diff
@@ -6,4 +6,23 @@
     <!-- TO DO : build profile page -->
     TEST PROFILE PAGE
 
+<!DOCTYPE html>
+    <h1>Profile Page</h1>
+    <h2>You are logged in as</h2>
+
+<!---------------------------------------profile display------------------------------------------------->    

```

Comment:
@aticodein please stick to standard HTML comments, only using two dashes.

`<!-- profile display -->`

Reply:
Changed to convention comment line.